test/Microsoft.TemplateEngine.TestHelper/TestUtils.cs

```diff
@@ -17,10 +17,28 @@ public static string CreateTemporaryFolder(string name = "")
         }
 
         public static string GetTestTemplateLocation(string templateName)
+        {
+            return GetTestArtifactLocation(Path.Combine("test_templates", templateName));
+        }
+
+        public static string GetTestNugetLocation(string templateName = "TestNupkgInstallTemplateV2.0.0.2.nupkg")
```

Comment:
I would suggest using https://github.com/dotnet/templating/blob/4cb831331dfc14ac1dd58b374f702f1b369ea7e3/test/dotnet-new3.UnitTests/DotnetNewInstantiate.cs#L189-L191

Reply:
Good to know.
I'll use the helper for installing. I'll still use the already packed artifact though (as packing takes several seconds)